### DISCUSSION TITLE
Support for types not in SQLite & FDW

### DISCUSSION THREAD

**Discussion by scherrey** [electric-sql/electric#1310] [General]
My understanding is that the local db is SQLite which implies only types that SQLite supports are available to the local app, correct? What about support for types like JSONB, GeoData, etc that are commonly available to Postgres but not natively present in SQLite?

Are capabilities like Postgres extensions, foreign data wrappers, etc impacted by ElectricSQL?

Looks like a promising technology and looking forward to trying it out.

**Comment by balegas** [comment]
SQLite has a very limited base type support. We handle the serialization of Postgres types on top of SQLite. We have a list of supported types here https://electric-sql.com/docs/usage/data-modelling/types

for other types that Postgres provides through extensions you can use them directly with PGlite instead of SQLite. We are currently working to load extensions inside the browser.

**Comment by KyleAMathews** [comment]
👋 we've been working the last month on a rebuild of the Electric server over at a temporary repo https://github.com/electric-sql/electric-next/

You can read more about why we made the decision at https://next.electric-sql.com/about

We're really excited about all the new possibilities the new server brings and we hope you'll check it out soon and give us your feedback.

We're now moving the temporary repo back here. As part of that migration we're closing all the old issues, PRs, and discussions. We really appreciate you taking the time to start this discussion! If it's still relevant in the new system, please reopen.
